lib/modules/apostrophe-personas-areas/public/js/editor.js

```diff
@@ -17,9 +17,9 @@ apos.define('apostrophe-areas-editor', {
       var data = apos.areas.getWidgetData($widget);
       var matches;
       var operator;
-      if (persona === 'none') {
+      /* if (persona === 'none') {
```

Comment:
Should be removed rather than commented, we can refer to commit history later if we need to see how it worked.